float32 distanceToFace
float32 angleToFace
std_msgs/Bool isPoster